.github/workflows/project_automation.yml

```diff
@@ -36,3 +29,15 @@ jobs:
         with:
           project-url: https://github.com/orgs/trezor/projects/58
           github-token: ${{ secrets.TREZOR_BOT_TOKEN }}
+
+  add-to-beta-project-suite-pull-requests:
+    name: Add new pull request to the Pull Requests project board
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@661f5254e8fcead08b054f5bc07b4d7f2af92c44
+        if: github.event_name == 'pull_request' && github.event.action == 'opened'
+        run: gh pr edit $PULL_REQUEST --add-project "Suite Pull Requests"
```

Comment:
I believe this line is extra, see https://github.com/actions/add-to-project for the documentation of the action
```suggestion
```

Reply:
Yes, that did the trick with the right syntax, but I am not sure it is working as expected, I cannot find this PR in the project, and it should be there, right? Can you find it @matejkriz ?

Reply:
The condition is `github.event.action == 'opened'` so it is not processing already opened PRs. Maybe will work if you close this PR and open a new one from the same commit. But I think we could merge it as it is.

Reply:
I see, :ok: